metricbeat/module/golang/heap/_meta/data.json

```diff
@@ -0,0 +1,46 @@
+{
+    "@timestamp": "2017-02-17T01:41:24.646Z",
+    "beat": {
+        "hostname": "MacBook-Pro.local",
+        "name": "MacBook-Pro.local",
+        "version": "6.0.0-alpha1"
+    },
+    "golang": {
+        "heap": {
+            "allocations": {
+                "active": 3325952,
+                "allocated": 2165152,
+                "total": 3393472,
+                "frees": 7975,
+                "idle": 1163264,
+                "mallocs": 27041,
+                "objects": 19066
+            },
+            "cmdline": "./metricbeat -httpprof=127.0.0.1:6060 -e -v",
+            "gc": {
+                "total_count": 1,
```

Comment:
here is not a group, it is only part of the name, it is not necessary to use dot, i think